### DISCUSSION TITLE
Dates hierarchy - quarters

### DISCUSSION THREAD

**Discussion by produktyn** [Profitbase/PowerBI-visuals-Gantt#2] [Ideas]
Please add option to show dates as quarters as that would be useful for planning purposes in many teams

**Comment by AndersAano** [comment]
Hi @produktyn, It's been a long time, but it's coming now!

![image](https://github.com/user-attachments/assets/a3b76bb7-0f63-45c4-aa95-d2ee1ce9cb23)

-Anders

**Comment by produktyn** [comment]
Hi, Great. Thank you for implementing this. Krzysztof

pt., 4 kwi 2025, 10:35 użytkownik Anders Aano ***@***.***>
napisał:

**Comment by AndersAano** [comment]
Hi @produktyn,  Should be available now.

Best regards
Anders
